src/Ledger/GovernanceActions.lagda

```diff
@@ -112,7 +183,48 @@ NeedsHash (TriggerHF _)        = GovActionID
 NeedsHash (ChangePParams _ _)  = GovActionID
 NeedsHash (TreasuryWdrl _)     = ⊤
 NeedsHash Info                 = ⊤
+\end{code}
 
+A governance action is an on-chain event that is triggered by a transaction, and there is a deadline after which
+the governance action cannot be enacted.
+\begin{itemize}
+\item
+  \defn{Ratification}. An action is said to be \defn{ratified} when it gathers enough votes in its favor (through the rules and parameters that are detailed
+  below).
+\item
+  \defn{Expiration}. An action that doesn't collect sufficient \AgdaField{yes} votes before its deadline is said to have \defn{expired}.
+\item
+  \defn{Enactment}. An action that has been ratified is said to be \defn{enacted} once it has been activated on the network.
+\end{itemize}
+
+
+\subsection{Voting and ratification}
+\label{sec:voting-and-ratification}
+A governance action is an on-chain event that is triggered by a transaction. There is a deadline after which
+the governance action cannot be enacted.
+\begin{itemize}
+\item
+  A governance action is said to be \defn{ratified} when it gathers enough votes in its favor
+  (through the rules and parameters that are detailed elsewhere [TODO: insert reference]).
+\item
+  A governance action is said to have \defn{expired} if it does not collect sufficient \AgdaField{yes} votes before the deadline.
+\item
+  An action that has been ratified is said to be \defn{enacted} once it has been activated on the network.
+\end{itemize}
+
+The data type \AgdaDatatype{Vote} represents the different voting options: \AgdaInductiveConstructor{yes}, \AgdaInductiveConstructor{no}, or \AgdaInductiveConstructor{abstain}.
+Each \AgdaField{vote} is recorded in a \AgdaRecord{GovVote} record along with the following meta-data:
```

Comment:
Only the `anchor` is metadata, everything else is regular data (since we use it in the rules).